openverse_api/catalog/settings.py

```diff
@@ -30,21 +30,28 @@
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY")
+SECRET_KEY = os.environ["DJANGO_SECRET_KEY"]  # required
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get("DJANGO_DEBUG_ENABLED", default=False) in true_strings
+DEBUG = os.getenv("DJANGO_DEBUG_ENABLED", "True").lower() in TRUE_STRINGS
```

Comment:
Unrelated to this PR, but should this security warning be codified somehow? It seems like we could write a condition that will crash the server or at least show a massive warning if this setting is on in production.

Reply:
With `decouple` all empty env vars without a default value make the application crash because it means it is required :) and also good catch here, I think the default value for `DEBUG` should remain `False` as if it were production.

Reply:
Changed `DEBUG` to be false by default in 62fb393.